### ISSUE TITLE
In Brave (Chromium based) the extension loads fonts with moz-extension: URLs, failing

### ISSUE THREAD

**Issue by Siderite** [prettierlichess/prettierlichess#163]
In Brave (Chromium based) the extension loads fonts with moz-extension: URLs, failing:

Refused to load the font 'moz-extension://epgnobcgnmchnhgkgpedebbmhbblfcob/fonts/RedHatText-Regular.ttf' because it violates the following Content Security Policy directive: "font-src 'self' lichess1.org".

**Comment by kieferro** [comment]
Thanks for pointing that out, but that's simply because the extension needs to work for both Firefox and Chrome, and trying to load the Firefox variant fails in Chromium-based browsers. However, this should not change anything about the functionality.

I guess there should also be a way to unify this. Is the message somehow annoying or is there a limitation in functionality? Otherwise, you can just ignore it.

**Comment by Siderite** [comment]
I am working on an extension as well and I have dev tools open all the time. And I see those errors, I think they're mine :D It doesn't affect functionality, but it is the only error in the console. BTW, great work on the extension.

**Comment by kieferro** [comment]
> And I see those errors, I think they're mine :D

I see 😄. I will see if I can do something about that without breaking anything.

> BTW, great work on the extension.

Thanks, glad you like it.

**Comment by kieferro** [comment]
@LofhJann Yes, maybe at some point we have to split the versions. But in any case where it can be avoided (and your PR shows that this is such a case - thanks), I would like to avoid it.
